front/app/components/EsriInitiativesMap/components/InitiativeInformation.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import eventEmitter from 'utils/eventEmitter';
+import InitiativePreview from './InitiativePreview';
+import CloseIconButton from 'components/UI/CloseIconButton';
+import styled from 'styled-components';
+
+type Props = {
+  modalPortalElement: HTMLDivElement;
+};
+
+const StyledCloseButton = styled(CloseIconButton)`
+  width: 100%;
+  justify-content: flex-end;
+  padding-top: 8px;
+  padding-right: 8px;
+`;
+
+const InitiativeInformation = ({ modalPortalElement }: Props) => {
+  const [selectedInitiativeId, setSelectedInitiativeId] = useState<
+    string | null
+  >(null);
+
+  // Listen for initiative selection event and set selected ID
+  useEffect(() => {
+    const subscription = eventEmitter
+      .observeEvent('initiativeSelectedEvent')
```

Comment:
I wasn't able to find another way to pass the currently selected initiative ID from the map component which didn't rerender the Esri map. I hope this will be alright?